README.md

```diff
@@ -223,10 +225,12 @@ dos-azul-lambda can be configured by setting a number of environment variables:
 * Set `DATA_BDL_DOCTYPE` to override the name of the ElasticSearch document type
   that dos-azul-lambda should expect to correspond with `DATA_BDL_INDEX`. By
   default, this is `databundle`.
-* Set `ES_HOST` to override the hostname of the ElasticSearch instance. By
-  default, this points to the `dss-azul-commons` ElasticSearch instance. This
-  variable should be specified without the protocol (i.e. without a leading
-  `http://` or `https://`).
+* Set `ES_HOST` to override the hostname of the ElasticSearch instance. On dev,
+  this points to the `dss-azul-commons` ElasticSearch instance
+  (`search-dss-azul-commons-lx3ltgewjw5wiw2yrxftoqr7jy.us-west-2.es.amazonaws.com`).
```

Comment:
I don't understand this. You removed the default host in app.py, but then here you say that the one that you removed is the default.

Or, phrased a different way, It looks to me like you're setting the hostname, not overriding it.